file-handling-example/src/download_file/handler.test.ts

```diff
@@ -0,0 +1,25 @@
+import { OperationHandlerTestSetup } from "@trayio/cdk-dsl/connector/operation/OperationHandlerTest";
+import { OperationHandlerResult } from "@trayio/cdk-dsl/connector/operation/OperationHandler";
+import { downloadFileHandler } from "./handler";
+import "@trayio/cdk-runtime/connector/operation/OperationHandlerTestRunner";
+
+OperationHandlerTestSetup.configureHandlerTest(
+	downloadFileHandler,
+	(handlerTest) =>
+		handlerTest
+			.usingHandlerContext("test")
+			.nothingBeforeAll()
+			.testCase("should download a file", (testCase) =>
+				testCase
+					.givenNothing()
+					.when(() => ({ file_path: '/cdk-testing/meme.jpeg' }))
```

Comment:
can we point to a file that exists?
people that will download the example won't notice that this file is not part of the repo?

Reply:
This is the file path in dropbox itself, not something local.

And if they run `upload_file` test first, then the file will exist

Reply:
Maybe add a comment then? Because I didn't know this before hand and this is an example repo?